integr8ly/installation
# INTLY-9063 cherrypick NodeCPUUtilitisation alerts

## Additional Information
https://issues.redhat.com/browse/INTLY-9063

## Verification Steps
As the verifier of the PR the following process should be done:

* Do an installation
* Verify the new alerts show in the Prometheus UI
* Run a pod(s) that uses a lot of cpu and/or memory
* Verify the Alerts fire as the node where the pod(s) are running are maxed out on cpu and/or memory

For upgrade:

* Install from v1.6.1 or v1.7.0
* The new alerts *shouldn't* exist in Prometheus UI
* Perform an upgrade
* Verify the new alerts show in the Prometheus UI

### Installation Verification
- Ensure the author of the PR has attached a log of the installation run from his branch to the jira or pr and check that it exited as expected.
- Verify the fresh installation is correct on cluster provided by PR author 
### Upgrade Verification
- After installation verification, notify the PR author to begin an upgrade on their cluster
- Ensure the developer of the PR has attached a log of the upgrade run from his branch to the jira or pr and check that it exited as expected. 
- If possible, look at the tasks that ran and see they match the PR
- Verify the upgrade is correct on cluster provided by PR author 



## Checklist



Is an upgrade task required and are there additional steps needed to test this?
